internal/api/delete_freight_v1alpha1.go

```diff
@@ -2,39 +2,58 @@ package api
 
 import (
 	"context"
+	"fmt"
 
 	"connectrpc.com/connect"
 	"github.com/pkg/errors"
```

Comment:
Curiosity question: what's the reason `github.com/pkg/errors` is extensively used throughout the code base while it has been marked as deprecated / archived, and Go's own error package has been quite enriched in more recent versions?

Reply:
Habit? I think some of the big improvements came in in 1.2 iirc and we haven't explored them adequately.

The subject of it being deprecated was discussed once before and judged a non-issue at the time given the relative simplicity of the package.

I'm unopposed to moving off of it as long as Go can wrap and unwind errors now.

btw... off topic... working on this PR, I think there is a lot more that can be done to improve how API requests are validated, but this PR is big enough already. Let's keep it in mind for another area for improvement when time permits. I'll open an issue when I'm not out of pocket.

Reply:
Wrapping is now possible using `fmt.Errorf("more context: %w"`, and the appropriate internal things are available to unwrap them. I will create an issue for it on Monday.

I will explore the API validation, it's an area I haven't got into yet although I noticed that _some_ errors were both caught by the CLI and the API while exploring other things.

Reply:
At one time, I used `fmt.Errorf` a lot. It used to not do anything special to make the error unwrappable. Just dug into source code and found this:

```golang
// If the format specifier includes a %w verb with an error operand,
// the returned error will implement an Unwrap method returning the operand.
// If there is more than one %w verb, the returned error will implement an
// Unwrap method returning a []error containing all the %w operands in the
// order they appear in the arguments.
```

Blame says special support for %w was added five years ago. Support for multiple %w was added 2 years ago. I missed the memo because I had an established habit of using `github.com/pkg/errors` by then.

Totally in favor of the switch when time permits.